src/components/DropdownTask.jsx

```diff
@@ -81,14 +95,41 @@ const DropdownTask = ({ columnId, taskId, handleEdit }) => {
                     </ListItemIcon>
                     <ListItemText primary="Edit" />
                 </MenuItem>
-                <MenuItem onClick={handleDelete}>
+                <MenuItem onClick={openSnackbar}>
                     <ListItemIcon>
                         <Delete fontSize="default" />
                     </ListItemIcon>
                     <ListItemText primary="Remove" />
                 </MenuItem>
+                <MenuItem>
+                    <ListItemIcon>
+                        <Archive fontSize="default" />
+                    </ListItemIcon>
+                    <ListItemText primary="Archive" />
+                </MenuItem>
             </Menu>
+            <Snackbar
+                classes={{ root: classes.snackbar }}
+                open={open}
+                anchorOrigin={{ horizontal: 'right', vertical: 'top' }}
+            >
+                <Alert variant="outlined" severity="error">
+                    <Grid item container direction="column">
+                        <Grid item>
```

Comment:
Lisäksi pikku ongelmana se, että noita voi klikkailla monta auki päällekkäin. Niin voisko tehdä sillai, että jos klikkaa jostain muualta kuin undo tai delete sillon kun se snackbar on auki nii se käsittelis sen Undo:na? eli sulkis ja sit ei pääsis tulee monta päälekkäin.